compiler/qsc_hir/src/global.rs

```diff
@@ -125,16 +153,16 @@ impl PackageIter<'_> {
                 });
 
                 Some(Global {
-                    namespace: Rc::clone(&namespace.name),
+                    namespace: namespace.into(),
                     name: Rc::clone(&name.name),
                     visibility: item.visibility,
                     status,
                     kind: Kind::Ty(Ty { id }),
                 })
             }
             (ItemKind::Namespace(ident, _), None) => Some(Global {
-                namespace: "".into(),
-                name: Rc::clone(&ident.name),
+                namespace: ident.into(),
+                name: "".into(),
```

Comment:
Do we want a bit more logic here? I would think that we'd want to be more explicit about namespaces that are "items" of other namespaces, such that Namespace globals would have their `name` field be the namespace name, and their `namespace` field be their parent namespace name.

Reply:
I'm satisfied with your double-checking this logic in response to me comment. Thanks for double checking this.